test/kitchen/site-cookbooks/dd-system-probe-check/recipes/default.rb

```diff
@@ -27,8 +27,15 @@
   end
 end
 
+file ::File.join(testdir, 'color_idx') do
+  content node[:color_idx].to_s
+  when !platform?('windows')
```

Comment:
I believe `when` is only a valid keyword in `case` statements in ruby, so this is causing a syntax error. I think this should be an `if` / `unless`:

```suggestion
  unless platform?('windows')
```

Reply:
Thanks. I forgot the `case` line, but I switched to `unless` as you suggested.